artifacts/fedora/fedora_test.go

```diff
@@ -98,6 +108,16 @@ var _ = Describe("Fedora", func() {
 						common.DefaultPreferenceEnv:   "fedora",
 					},
 				},
+				&fedora{
+					Version: "35",
+					Arch:    "aarch64",
```

Comment:
the same

Reply:
We don't have constants of other values, why this one should be?